autospec/specfiles.py

```diff
@@ -1008,11 +1005,11 @@ def write_configure_pattern(self):
             self._write_strip("unset PKG_CONFIG_PATH")
             self._write_strip("pushd ../buildavx2/" + self.config.subdir)
             self.write_build_prepend()
-            self._write_strip("export CFLAGS=\"$CFLAGS -m64 -march=x86-64-v3 -Wl,-z,x86-64-v3\"")
-            self._write_strip("export CXXFLAGS=\"$CXXFLAGS -m64 -march=x86-64-v3 -Wl,-z,x86-64-v3\"")
-            self._write_strip("export FFLAGS=\"$FFLAGS -m64 -march=x86-64-v3 -Wl,-z,x86-64-v3\"")
-            self._write_strip("export FCFLAGS=\"$FCFLAGS -m64 -march=x86-64-v3\"")
-            self._write_strip("export LDFLAGS=\"$LDFLAGS -m64 -march=x86-64-v3\"")
+            self._write_strip('CFLAGS="$CLEAR_INTERMEDIATE_CFLAGS -m64 -march=x86-64-v3 -Wl,-z,x86-64-v3 "')
```

Comment:
doesn't this need an export ?

Reply:
Not from my testing looking at the V3 vs 32bit files that I was poking for this change.